### PR TITLE
Added authorizationFallbackResponse to demo article

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: views/amp-access/article.html [hunk 1]
```diff
@@ -79,7 +83,7 @@
     <main role="main">
     <article>
 
-      <section id="meter-notif" amp-access="access" class="meter-section" data-z-depth="2">
+      <section id="meter-notif" amp-access="access OR error" class="meter-section" data-z-depth="2">
```
Comment (sebastianbenz) [inline views/amp-access/article.html]: This section is visible because we recommend to show content in case of an error?

**Comment by sebastianbenz** [comment]
LGTM - one question

File: views/amp-access/article.html (hunk 1 above)
Comment (andreban) [inline views/amp-access/article.html]: The purpose is just to demo how one can use the fallback do show or hide a section if an error occurs. I don't think we have a recommendation around this at the moment.

Comment (sebastianbenz) [inline views/amp-access/article.html]: I that case I'd recommend not adding it in to the content section - I might give the wrong ideas. Having one separate section for it should be enough.

**Pull request merged**